css/css-text-decor/parsing/text-decoration-valid.html

```diff
@@ -18,6 +18,8 @@
 
 test_valid_value("text-decoration", "double overline underline", "underline overline double");
 test_valid_value("text-decoration", "underline overline line-through red");
+test_valid_value("text-decoration", "underline overline line-through blink", ["underline overline line-through", "underline overline line-through blink"]);
+test_valid_value("text-decoration", "underline overline line-through blink red", ["underline overline line-through rgb(255, 0, 0)", "underline overline line-through blink rgb(255, 0, 0)"]);
```

Comment:
`rbg(255, 0, 0)` should be `red` because it's not a computed value.

Both of Firefox and Google Chrome return `underline overline line-through blink red`.

Reply:
@vitorroriz will fix this, thanks for pointing this out!

Reply:
This was fixed here: https://github.com/web-platform-tests/wpt/pull/54240